src/modules/bookmark/utils.js

```diff
@@ -26,17 +25,25 @@ export const validateBookmarks = (data) => {
 export const getIndexOfBookmark = (bookmarks, { address, token = tokenMap.LSK.key }) =>
   bookmarks[token].findIndex((bookmark) => bookmark.address === address);
 
+export const getIndexOfLabel = (bookmarks, { label, token = tokenMap.LSK.key }) =>
```

Comment:
why export a function when its not re used anywhere?

Reply:
Fixed